### ISSUE TITLE
[Kubernetes] Request for help with updating docs for Dockershim removal

### ISSUE THREAD

**Issue by mrbobbytables** [cncf/techdocs#88]
In the upcoming (April 2022) Kubernetes 1.24 release, support for [Dockershim](https://kubernetes.io/blog/2020/12/02/dockershim-faq/), the native mechanism to talk to the Docker Container Runtime will be removed. From both surveys that the [Kubernetes project](https://kubernetes.io/blog/2021/11/12/are-you-ready-for-dockershim-removal/) has run and those from [other groups](https://www.datadoghq.com/container-report/#8) have shown that many end users still using Dockershim and are ill-prepared for the migration.

The Kubernetes project would like to request assistance with helping update docs for this upcoming change. Scope may expand as things are discovered but right now it would include:
- Update current documentation with non-docker centric commands (e.g. replace docker with ctr and crictl)
- Improve [migration guide](https://kubernetes.io/docs/tasks/administer-cluster/migrating-from-dockershim/) and expand on pitfalls, gotchas, and things to avoid
- Assemble and publish links to 3rd party migration docs (e.g. [telemetry](https://docs.google.com/document/d/1ZFi4uKit63ga5sxEiZblfb-c23lFhvy6RXVPikS8wf0/edit#heading=h.9d9v9i7zhdj2))
- (Optional) Help other CNCF projects update their own docs (e.g. containerd, CRI-O, and Falco) with better documentation and migration guides

If I can provide any further information, please let me know.

Thanks!

**Comment by justaugustus** [comment]
FYI Kubernetes SIG Release leads (to track):
@saschagrunert @jeremyrickard @cpanato @puerco

**Comment by celestehorgan** [comment]
@mrbobbytables the CNCF techdocs team will help with the following pieces specifically:

- owning looking through the existing kubernetes.io content, identifying what needs to be updated, and drafting github issues for them 
- working with the technical teams (SIG Node/SIG Arch) to identify replacements where they aren't obvious 
- communicating with other CNCF projects 
- reaching out to the end user community to prepare them for this change 
- overall project management for the duration of the 1.24 release.

We expect to work in tandem with the Kubernetes project community on the following pieces:
- executing on and approving the changes in Kubernetes docs: while we expect to take on some of these changes, we suspect the magnitude of the work to be done will be a collaborative effort by the community 
- identifying appropriate reviewers for changes and ensuring that technical review is performed on each change
- handling any long tail that comes out of this release.

In short: the techdocs team will provide staffing for the duration of the 1.24 release to ensure that the project gets over this critical hump; but we absolutely expect help from the rest of the project as well :)

Regards,
Celeste

**Comment by SergeyKanzhelev** [comment]
> * we suspect the magnitude of the work to be done will be a collaborative effort by the community

one of the issue is that we had tasks like https://github.com/kubernetes/website/issues/25879 filed for a long time with very little contributors interest. One reason may be that this approach in changing runtimes is unlikely to be used by managed k8s deployments, so there may be fewer people familiar deeply with this. Help with those items from CNCF would be greatly appreciated by end users. Is it something that can be sponsored by CNCF as well?

**Comment by nate-double-u** [comment]
Can we close this having completed the v1.24 release @mrbobbytables?
